formidable/forms/conditions.py

```diff
@@ -12,6 +12,13 @@
   See :class:`formidable.forms.conditions.DisplayIffCondition`
 - Conditional access: change access level (required/editable/...)
 
+.. Note::
+    At this moment, we can guarantee only the support of the checkboxes and
+    dropdown lists, but normally you could use it for any type you want.
+    You could specify types allowed for the conditions using settings variable
+    ``FORMIDABLE_CONDITION_FIELDS_ALLOWED_TYPES``
```

Comment:
It'd be great to add here:

* one or more examples of usage,
* give the list of the available types.

Reply:
And I think it'd be better to have a proper section in the docs, rather than a note in the docstring. This is a configuration option, and it should be explained along with the other settings available.
Also, this new option should be mentioned in the changelog entry.